src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerControllerTests.cs

```diff
@@ -448,6 +448,10 @@ public async Task ServiceOfNoticeDetails_Get_ReturnViewModelAndMapsData()
 
             model.Should().NotBeNull();
             model.ServiceOfNoticeViewModel.Should().NotBeNull();
```

Comment:
@stevelee-civica can you setup 
 A.CallTo(() => mapper
                            .Map<SmallProducerSubmissionMapperData, ServiceOfNoticeViewModel>(A<SmallProducerSubmissionMapperData>.That.Matches(sd => sd.SmallProducerSubmissionData.Equals(controller.SmallProducerSubmissionData))))
                            
                            
to return a ServiceOfNoticeViewModel and check that model.ServiceOfNoticeViewModel.Should().be(the expected value)

Reply:
updated